.pre-commit-config.yaml

```
@@ -19,6 +19,7 @@ repos:
   rev: v2.3.0
   hooks:
   - id: codespell
+    args: ["--ignore-words", "doc/styles/config/vocabularies/ANSYS/accept.txt", "-w"]
```

Isn't this config coming from the pyproject.toml?...

---

For some reason it isn't respecting what's in the pyproject.toml, so I had to include that for it to work

---

I have faced the same issue in other projects. Could you try to add this?

```suggestion
    args: ["--ignore-words", "doc/styles/config/vocabularies/ANSYS/accept.txt", "-w"]
    additional_dependencies:
      - tomli
```

Even with that, it is not working in other repos. Maybe things have changed...